projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -304,6 +304,10 @@ public function set_up_sharing_limits() {
 		$connections      = $publicize->get_all_connections();
 		$shares_remaining = $info['shares_remaining'];
 
+		if ( empty( $connections ) ) {
```

Comment:
@pablinos I think enforcing it there is the way to go, but I was worried about side effects which is why I opted for the check here. I can do some research and see if changing it could cause trouble anywhere.